examples/rvc-app/rvc-common/src/rvc-service-area-delegate.cpp

```diff
@@ -25,8 +26,8 @@ using namespace chip::app::Clusters::ServiceArea;
 CHIP_ERROR RvcServiceAreaDelegate::Init()
 {
     // hardcoded fill of SUPPORTED MAPS for prototyping
-    uint8_t supportedMapId_XX = 3;
-    uint8_t supportedMapId_YY = 245;
+    uint32_t supportedMapId_XX = 3;
```

Comment:
This doesn't seem like a prototyping file since it's in common - is this delegate meant to be replaced in a real product?

Reply:
@cecille yes, this delegate must be replaced with something that uses real data in an actual device